src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -79,6 +80,13 @@ export class IntegrationListDirectory extends AsyncComponent<
     };
   }
 
+  componentDidMount() {
```

Comment:
I didn't have a chance to document this quite yet in the `experimentConfig` but you would have an easier time here using the `withExperiment` HoC, since then you won't have to worry about calling `logExperiment`.

@scefali I'm surprised you didn't catch this, since when you removed the previous experiment in here, it was using the HoC haha